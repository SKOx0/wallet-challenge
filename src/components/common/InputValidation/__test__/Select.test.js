import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SelectValidation from '../Select';

describe('<SelectValidation />', () => {
  it('deve ser igual ao snapshot', () => {
    const renderedComponent = shallow(<SelectValidation />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
