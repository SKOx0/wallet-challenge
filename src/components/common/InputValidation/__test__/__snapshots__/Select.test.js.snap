// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectValidation /> deve ser igual ao snapshot 1`] = `
.emotion-0 {
  height: 50px;
  padding: 15px;
  width: 100%;
  border-radius: 3px;
  border-width: 1px;
  border-color: #D2D2D2;
  font-size: 1.0rem;
}

.emotion-0:focus {
  outline: none;
}

.emotion-0[readonly],
.emotion-0 [disabled] {
  opacity: 0.5;
}

<SelectValidation
  className="emotion-0"
/>
`;
